webpack/app/js/layout.js

```diff
@@ -355,12 +356,11 @@ export default class LayoutView {
       'top': '0',
       'width': '15px'
     });
-    let onClick = function() {
-      let element = button.parent('div');
+    button.click(function() {
+      var element = $(this).parent('div');
       element.remove();
-      this.$le.trigger('modified.layout');
-    };
-    button.on('click', onClick.bind(this));
+      le_delete.trigger('modified.layout');
+    });
```

Comment:
please use the `bind this` in the event instead to fix the this inside the jquery event. ex.:

https://github.com/collective/collective.cover/blob/master/webpack/app/js/cssclasswidget.js#L19

this way you don't need the `le_delete` variable.

also we are using ES6 here, prefer to use `let` and `const` instead of `var` to create variables; prefer to use arrow functions instead of old functions if possible.

please take a looks at http://es6-features.org